.metadata-tools/metadataanalyzer/go.mod

```diff
@@ -0,0 +1,199 @@
+module metadataanalyzer
+
+go 1.20
+
+replace github.com/dapr/components-contring => ../../
```

Comment:
components-contring -> components-contrib ?

Reply:
Please use the suggestion feature in the future so it can be applied @tmacam

Reply:
```suggestion
replace github.com/dapr/components-contrib => ../../
```